workflow/react-project/src/routing/index.jsx

```diff
@@ -0,0 +1,20 @@
+import MainPage from '@pages/MainPage';
+import SearchPage from '@pages/SearchPage';
+
+export const routes = [
+  {
+    path: '/mainPage',
+    element: <MainPage />,
+    key: '1',
+  },
+  {
+    path: '/search',
+    element: <SearchPage />,
+    key: '2',
+  },
+  {
+    path: '/*',
+    element: <MainPage />,
+    key: '3',
+  },
+];
```

Comment:
you can change to `Object`, and use it to set paths to buttons.
`/mainPage` -> usr just `/`